internal/supervisor/installation.go

```diff
@@ -172,12 +174,43 @@ func NewInstallationSupervisor(
 }
 
 // NewInstallationSupervisorSchedulingOptions creates a new InstallationSupervisorSchedulingOptions.
-func NewInstallationSupervisorSchedulingOptions(balanceInstallations bool, clusterResourceThreshold, clusterResourceThresholdScaleValue int) InstallationSupervisorSchedulingOptions {
-	return InstallationSupervisorSchedulingOptions{
+func NewInstallationSupervisorSchedulingOptions(balanceInstallations bool, clusterResourceThreshold, thresholdCPUOverride, thresholdMemoryOverride, thresholdPodCountOverride, clusterResourceThresholdScaleValue int) InstallationSupervisorSchedulingOptions {
```

Comment:
Since there is a bunch of logic now, I would add unit test for it. There is a lot of ints with similar names so it easy to slip up.

Reply:
Yep, good point. Tests added.